libs/auth/src/lib/components/register/register.component.ts

```diff
@@ -36,6 +41,46 @@ export class RegisterComponent {
         }
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Hier bitte nicht die eslint Warnung suppressen sondern void als return Type angeben. Außerdem bitte die Methode onKey() mit einem Zugriffsmodifikator (in dem Fall public) versehen

Reply:
Ist angepasst